pkg/patterns/declarative/reconciler.go

```diff
@@ -166,53 +178,63 @@ func (r *Reconciler) BuildDeploymentObjects(ctx context.Context, name types.Name
 	log := log.Log
 
 	// 1. Load the manifest
-	manifestStr, err := r.loadRawManifest(ctx, instance)
+	manifestFiles, err := r.loadRawManifest(ctx, instance)
 	if err != nil {
 		log.Error(err, "error loading raw manifest")
 		return nil, err
 	}
-
+	mainfestObjects := &manifest.Objects{}
 	// 2. Perform raw string operations
-	for _, t := range r.options.rawManifestOperations {
-		transformed, err := t(ctx, instance, manifestStr)
+	for filepath, manifestStr := range manifestFiles {
```

Comment:
suggest a different name for this since it shadows the stdlib package (sys/filepath), perhaps `manifestPath`